<!--
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonCerrar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar sesión</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRegistrar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGUsu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar usuario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGMon">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar monitor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGInv">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar inventario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGAdm">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar administrativo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
-->

<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel principal - Administrador</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: #2e2e2e;
        color: white;
    }
    QPushButton {
        background-color: white;
        border-radius: 10px;
        font-size: 16px;
        padding: 10px;
    }
    QPushButton:hover {
        background-color: #dddddd;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">

   <widget class="QLabel" name="labelIcono">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>10</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap>:/imagenes/gimnasio.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButtonCerrar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar sesión</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButtonRegistrar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar usuario</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButtonGUsu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar usuario</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButtonGMon">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar monitor</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButtonGInv">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar inventario</string>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButtonGAdm">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>600</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestionar administrativo</string>
    </property>
   </widget>

  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
